include/zephyr/bluetooth/audio/bap.h

```diff
@@ -317,6 +317,15 @@ struct bt_bap_scan_delegator_recv_state {
 };
 
 struct bt_bap_scan_delegator_cb {
+
+	/**
+	 * @brief Receive states full notification
```

Comment:
Please elaborate on this: This callback is in fact not called when the receive states are full (probably also a new name), but rather when a new source is attempted to be added and is failing because there are no more room.